nselib/httpspider.lua

```diff
@@ -971,6 +1000,10 @@ Crawler = {
       self.options.withindomain = false
     end
 
+    if self.options.docookies == 0 then
```

Comment:
You can set to nil the object when using stdnse.get_script_args() like:
local opt = stdnse.get_script_args() or nil